modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-service/src/main/java/com/liferay/portal/workflow/kaleo/forms/internal/instance/lifecycle/AddDefaultKaleoProcessStructuresPortalInstanceLifecycleListener.java

```diff
@@ -62,12 +60,6 @@ public void portalInstanceRegistered(Company company) throws Exception {
 	@Reference
 	private GroupLocalService _groupLocalService;
 
-	@Reference
-	private KaleoProcessLocalService _kaleoProcessLocalService;
-
-	@Reference(target = ModuleServiceLifecycle.PORTAL_INITIALIZED)
-	private ModuleServiceLifecycle _moduleServiceLifecycle;
```

Comment:
Do not remove `ModuleServiceLifecycle.PORTAL_INITIALIZED` it's used to ensure the component will be activated after the portal is initialized.

Thanks!